packages/expo-network/android/src/main/java/expo/modules/network/NetworkModule.kt

```diff
@@ -91,6 +83,50 @@ class NetworkModule : Module() {
       get() = this.value != "NONE" && this.value != "UNKNOWN"
   }
 
+  private fun emitNetworkState() {
+    val networkState = fetchNetworkState()
+    sendEvent(NETWORK_STATE_EVENT_NAME, networkState)
+  }
+
+  private fun fetchNetworkState(): Bundle {
+    val result = Bundle()
+    val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
```

Comment:
We do this 3 times. How about we make this a property on the module with a getter
```kotlin
val connectivityManager get() = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager

```